src/components/learner-credit-management/BudgetDetailPageHeader.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  Row, Col, Breadcrumb, Stack,
+} from '@edx/paragon';
+
+import { ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+import { useBudgetId, useSubsidyAccessPolicy } from './data';
+
+const BudgetDetailPageHeader = ({ enterpriseSlug }) => {
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const budgetDisplayName = subsidyAccessPolicy?.displayName || 'Overview';
```

Comment:
[question] Not sure if this worth exploring to grab a different value other than `display_name` since most of our subsidies have `display_name` as `null`

Reply:
While most budgets (i.e., subsidy access policies) have `display_name=null` today, the expectation is that the provisioning tool in the support-tools MFE will be setting the `display_name` field for net-new budgets. For now, we are relying on ensuring `display_name` is manually set for customers via Django Admin and falling back to "Overview" similar to what the budget overview page does today.